charts/kafka-lag-exporter/templates/030-ConfigMap.yaml

```diff
@@ -76,6 +76,40 @@ data:
         }
         {{- end }}
       ]
+      {{- if .Values.redis.enabled }}
+      redis = {
+        {{- with .Values.redis.enabled }}
+        enabled = {{ . | quote }}
```

Comment:
I don't think it's necessary to quote all of these values. It should only be necessary when strings contain YAML reserved characters.

Reply:
Probably safer, but I removed all the `with` statements.